src/main/java/de/tum/in/www1/artemis/web/rest/admin/AdminCourseResource.java

```diff
@@ -126,4 +135,21 @@ public ResponseEntity<Void> deleteCourse(@PathVariable long courseId) {
         courseService.delete(course);
         return ResponseEntity.ok().headers(HeaderUtil.createEntityDeletionAlert(applicationName, true, Course.ENTITY_NAME, course.getTitle())).build();
     }
+
+    /**
+     * Helper function, that's creates a channel within a course
+     *
+     * @param course         the course, where the channel should be created
+     * @param name           the name of the channel
+     * @param isAnnouncement whether the channel is a announcement channel
+     */
+    private void createChannel(Course course, String name, Boolean isAnnouncement) {
```

Comment:
```suggestion
    private void createChannel(Course course, String name, boolean isAnnouncement) {
```
Additionally wouldn't it be enough to include the check for the announcement channel inside this method?

Reply:
We removed the parameter